dask/array/tests/test_reductions.py

```diff
@@ -407,8 +407,14 @@ def test_nan_object(func):
 
         if func in {"nanmin", "nanmax"}:
             warnings.simplefilter("default", RuntimeWarning)
+        elif func in {"min", "max"}:
+            warnings.simplefilter("ignore", RuntimeWarning)
 
         assert_eq(getattr(np, func)(x, axis=0), getattr(da, func)(d, axis=0))
```

Comment:
I suspect what you're doing here is ignoring the warnings, running this line, then re-enabling the warnings. I would prefer if these were grouped around the test line, not conflated with the previous clause re-enabling warnings for `nanmin`/`nanmax`.

```
if func in {"min", "max"}:
    warnings.simplefilter("ignore", RuntimeWarning)
assert_eq(getattr(np, func)(x, axis=0), getattr(da, func)(d, axis=0))
if func in {"min", "max"}:
    warnings.simplefilter("ignore", RuntimeWarning)
```

Reply:
Done.